Version after 4.1.2 added a dependency on tslib?

After upgrading from 4.1.2 -> 4.1.9 of @promster/express, my application encountered the following error:

```
internal/modules/cjs/loader.js:796
    throw err;
    ^
Error: Cannot find module 'tslib'
Require stack:
- /usr/src/app/node_modules/@promster/express/dist/modules/middleware/middleware.js
- /usr/src/app/node_modules/@promster/express/dist/modules/middleware/index.js
- /usr/src/app/node_modules/@promster/express/dist/modules/index.js
- SNIP
- SNIP
- /usr/src/app/src/app.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:793:17)
    at Function.Module._load (internal/modules/cjs/loader.js:686:27)
    at Module.require (internal/modules/cjs/loader.js:848:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/usr/src/app/node_modules/@promster/express/dist/modules/middleware/middleware.js:3:17)
    at Module._compile (internal/modules/cjs/loader.js:955:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
    at Module.load (internal/modules/cjs/loader.js:811:32)
    at Function.Module._load (internal/modules/cjs/loader.js:723:14)
    at Module.require (internal/modules/cjs/loader.js:848:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/usr/src/app/node_modules/@promster/express/dist/modules/middleware/index.js:3:22)
    at Module._compile (internal/modules/cjs/loader.js:955:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
    at Module.load (internal/modules/cjs/loader.js:811:32)
    at Function.Module._load (internal/modules/cjs/loader.js:723:14) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/src/app/node_modules/@promster/express/dist/modules/middleware/middleware.js',
    '/usr/src/app/node_modules/@promster/express/dist/modules/middleware/index.js',
    '/usr/src/app/node_modules/@promster/express/dist/modules/index.js',
    SNIP,
    SNIP,
    SNIP
  ]
```

`tslib` is not listed as a dependency in https://github.com/tdeekens/promster/blob/master/packages/express/package.json, but in the compiled code under node_modules/@promster/express/dist/modules/middleware/middleware.js, there is

```
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const tslib_1 = require("tslib");
const http_1 = require("http");
```

I can work around this by requiring `tslib` in my own application, but shouldn't @promster/express require its own dependencies?

--- a/packages/express/package.json
+++ b/packages/express/package.json
@@ -39,7 +39,8 @@
   ],
   "dependencies": {
     "@promster/metrics": "^4.1.8",
-    "merge-options": "2.0.0"
+    "merge-options": "2.0.0",
+    "tslib": "1.11.1"
   },
   "devDependencies": {
     "@types/express": "4.17.6"

--- a/packages/hapi/package.json
+++ b/packages/hapi/package.json
@@ -40,7 +40,8 @@
   "dependencies": {
     "@promster/metrics": "^4.1.8",
     "merge-options": "2.0.0",
-    "semver": "7.3.2"
+    "semver": "7.3.2",
+    "tslib": "1.11.1"
   },
   "devDependencies": {
     "@promster/types": "^1.0.3",

--- a/packages/marblejs/package.json
+++ b/packages/marblejs/package.json
@@ -40,7 +40,8 @@
   "dependencies": {
     "@promster/metrics": "^4.1.8",
     "merge-options": "2.0.0",
-    "rxjs": "^6.5.3"
+    "rxjs": "^6.5.3",
+    "tslib": "1.11.1"
   },
   "devDependencies": {
     "@marblejs/core": "3.1.0",

--- a/packages/metrics/package.json
+++ b/packages/metrics/package.json
@@ -44,7 +44,8 @@
     "optional": "0.1.4",
     "ts-essentials": "6.0.4",
     "url": "0.11.0",
-    "url-value-parser": "2.0.1"
+    "url-value-parser": "2.0.1",
+    "tslib": "1.11.1"
   },
   "devDependencies": {
     "@promster/types": "^1.0.3",

--- a/packages/server/package.json
+++ b/packages/server/package.json
@@ -38,7 +38,8 @@
     "prometheus"
   ],
   "dependencies": {
-    "@promster/metrics": "^4.1.8"
+    "@promster/metrics": "^4.1.8",
+    "tslib": "1.11.1"
   },
   "devDependencies": {
     "@types/node": "13.13.4"

